pkg/apis/core/types_shoot.go

```diff
@@ -705,6 +707,19 @@ type Volume struct {
 	Encrypted *bool
 }
 
+// CRI contains information about the Container Runtimes.
+type CRI struct {
+	// The name of the CRI library
+	Name CRIName
```

Comment:
Can we also already add an optional `providerConfig` field like for other extension types:

https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/types_shoot.go#L251-L253